playground/providers.ts

```diff
@@ -694,5 +694,36 @@ export const providers: Provider[] = [
         quality: 75
       }
     ]
+  },
+  // Directus
+  {
+    name: 'directus',
+    samples: [
+      {
+        src: 'ad514db1-eb90-4523-8183-46781437e7ee',
+        alt: 'Image 1'
+      },
+      {
+        src: 'ad514db1-eb90-4523-8183-46781437e7ee.jpg',
+        alt: '1024px width',
+        width: 1024,
+        height: 256,
+        fit: 'cover',
+        modifiers: { withoutEnlargement: 'true' }
+      },
+      {
+        src: 'ad514db1-eb90-4523-8183-46781437e7ee',
+        alt: '256px width, webp',
+        width: 256,
+        format: 'webp'
+      },
+      {
+        src: 'ad514db1-eb90-4523-8183-46781437e7ee',
+        alt: '256px width, webp',
+        width: 256,
+        format: 'webp',
+        modifiers: { withoutEnlargement: 'true', transforms: '[["blur", 4], ["negate"]]' }
```

Comment:
these modifiers don't seem to be showing up in the URLs loaded in the e2e test snapshots

Reply:
ahh, I see - you may have added these manually.

in future, you can run `pnpm vitest run -u` to update snapshots

Reply:
Yes I've added them manually, sorry. First time learning vitest

>in future, you can run `pnpm vitest run -u` to update snapshots

ty